include/contest.h

```diff
@@ -408,30 +424,31 @@ struct ContestResourcesField20
 
 struct ContestResources
 {
-    struct Contest *field_0;
-    struct ContestantStatus *field_4;
+    struct Contest *contest;
+    struct ContestantStatus *status;
     struct UnknownContestStruct7 *field_8;
-    struct ContestAIInfo *field_C;
+    struct ContestAIInfo *aiData;
     struct UnknownContestStruct5 *field_10;
     struct UnknownContestStruct4 *field_14;
     struct ContestStruct_field_18 *field_18;
     struct ContestResourcesField1C * field_1c;
     struct ContestResourcesField20 * field_20;
-    u8 * field_24[4];
+    u8 * contestBgTilemaps[CONTESTANT_COUNT];
     void * field_34;
     void * field_38;
     void * field_3c;
 };
 
-#define sContest (*gContestResources->field_0)
-#define sContestantStatus (gContestResources->field_4)
-#define shared192D0 (*gContestResources->field_8)
-#define eContestAI (gContestResources->field_C)
-#define shared19328 (*gContestResources->field_10)
-#define shared19338 (*gContestResources->field_14)
-#define shared15800 (gHeap + 0x18000)
-#define shared16800 (gHeap + 0x19000)
-#define shared18000 (*(struct Shared18000 *)(gHeap + 0x1a000))
+#define eContest (*gContestResources->contest)
+#define eContestantStatus (gContestResources->status)
+#define eContestResources8 (*gContestResources->field_8)
+#define eContestAI (*gContestResources->aiData)
+#define eContestResources10 (*gContestResources->field_10)
+#define eContestResources14 (*gContestResources->field_14)
+#define eUnzippedContestAudienceGfx (gHeap + 0x18000)
+#define eUnknownHeap19000 (gHeap + 0x19000)
+#define eContestDebugMode (gHeap[0x1a000])
```

Comment:
Why is this bracketed when the others aren't?

Reply:
Because it's a value, not a pointer.